### PR TITLE
Added Info - University of Moratuwa

### PR DESCRIPTION
**Pull request by suthagar23**
Added university Info and one faculty information with some departments information. 
Issue : #2

### CODE REVIEW

File: SriLanka/UniversityofMoratuwa/faculties_departments.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                    "web_url" : "http://www.tlm.mrt.ac.lk",
+                    "hod_info" : {
+                        "name" : "Mr.S.N. Bentotage",
+                        "email" : "head-tlm@uom.lk"
+                    }
+                }
+            ]
+        }
+    ]
+}
```
Comment (inncaps) [inline SriLanka/UniversityofMoratuwa/faculties_departments.json]: New line should be added after the file

File: SriLanka/UniversityofMoratuwa/university_info.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            "email" : "dvc@uom.lk",
+            "contact_no" : "+94 3139398"
+        },
+        "registrar"  : {
+            "name" : "Mr. A.L.J. Sadique",
+            "email" : "registrar@uom.lk",
+            "contact_no" : "+94 11 2651610"
+        }
+    }
+}
```
Comment (inncaps) [inline SriLanka/UniversityofMoratuwa/university_info.json]: New line should be added after the file

File: SriLanka/UniversityofMoratuwa/faculties_departments.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                        "name" : "Bandara J.M.S.J",
+                        "email" : "bandara@civil.mrt.ac.lk",
+                        "contact_no" : "+94-11-2650567 ( Ext : 2129 )"
+                    }
+                },
+                {
+                    "department_name" : "Department of Computer Science & Engineering",
+                    "web_url" : "http://www.cse.mrt.ac.lk/",
+                    "hod_info" : {
+                        "name" : "DR. A. SHEHAN PERERA",
```
Comment (inncaps) [inline SriLanka/UniversityofMoratuwa/faculties_departments.json]: Can you please change this to Camel case

File: SriLanka/UniversityofMoratuwa/faculties_departments.json (hunk 1 above)
Comment (suthagar23) [reply SriLanka/UniversityofMoratuwa/faculties_departments.json]: Fixed

File: SriLanka/UniversityofMoratuwa/university_info.json (hunk 2 above)
Comment (suthagar23) [reply SriLanka/UniversityofMoratuwa/university_info.json]: Fixed

File: SriLanka/UniversityofMoratuwa/faculties_departments.json (hunk 3 above)
Comment (suthagar23) [reply SriLanka/UniversityofMoratuwa/faculties_departments.json]: Fixed

**Pull request merged**